frontend/src/component/operator-wizard/OperatorWizard.js

```diff
@@ -350,7 +355,42 @@ export default class OperatorWizard extends Component {
   getJsonSchemaPathForYaml(jsonPath) {
     return jsonPath.slice(2, jsonPath.length);
   }
+  addExternalDBEnvVars(jsonObject) {
+    const tempEnv = [
+      { name: EXTENSIONS_IMAGE_KEY, value: EXTENSIONS_IMAGE_VALUE },
+      {
+        name: EXTENSIONS_IMAGE_NAMESPACE_KEY,
+        value: EXTENSIONS_IMAGE_NAMESPACE_VALUE
+      }
+    ];
+
+    var server;
+    var obj = Dot.object(jsonObject);
```

Comment:
Minor: We may want to change "obj" to something meaningful?

Reply:
updated